server/config.js

```diff
@@ -30,7 +30,7 @@ const config = {
             }
         },
         server: {
-            WORKFLOW_SERVICE: process.env.WORKFLOW_SERVICE || 'http://localhost:8081',
+            WORKFLOW_SERVICE: process.env.WORKFLOW_SERVICE || 'http://localhost:8091',
```

Comment:
Is there a reason for changing the default port, does something else run on 8081?